crates/orchestrator/src/queue/sqs/mod.rs

```diff
@@ -6,45 +6,47 @@ use aws_sdk_sqs::types::QueueAttributeName;
 use aws_sdk_sqs::Client;
 use color_eyre::eyre::eyre;
 use color_eyre::Result;
-use lazy_static::lazy_static;
 use omniqueue::backends::{SqsBackend, SqsConfig, SqsConsumer, SqsProducer};
 use omniqueue::{Delivery, QueueError};
-use utils::env_utils::get_env_var_or_panic;
-
-use crate::queue::job_queue::{
-    DATA_SUBMISSION_JOB_PROCESSING_QUEUE, DATA_SUBMISSION_JOB_VERIFICATION_QUEUE, JOB_HANDLE_FAILURE_QUEUE,
-    PROOF_REGISTRATION_JOB_PROCESSING_QUEUE, PROOF_REGISTRATION_JOB_VERIFICATION_QUEUE, PROVING_JOB_PROCESSING_QUEUE,
-    PROVING_JOB_VERIFICATION_QUEUE, SNOS_JOB_PROCESSING_QUEUE, SNOS_JOB_VERIFICATION_QUEUE,
-    UPDATE_STATE_JOB_PROCESSING_QUEUE, UPDATE_STATE_JOB_VERIFICATION_QUEUE, WORKER_TRIGGER_QUEUE,
-};
+use serde::Serialize;
+
+use super::QueueType;
 use crate::queue::{QueueConfig, QueueProvider};
 use crate::setup::SetupConfig;
 
-pub struct SqsQueue;
-
-lazy_static! {
-    /// Maps Queue Name to Env var of queue URL.
-    pub static ref QUEUE_NAME_TO_ENV_VAR_MAPPING: HashMap<&'static str, &'static str> = HashMap::from([
-        (DATA_SUBMISSION_JOB_PROCESSING_QUEUE, "SQS_DATA_SUBMISSION_JOB_PROCESSING_QUEUE_URL"),
-        (DATA_SUBMISSION_JOB_VERIFICATION_QUEUE, "SQS_DATA_SUBMISSION_JOB_VERIFICATION_QUEUE_URL"),
-        (PROOF_REGISTRATION_JOB_PROCESSING_QUEUE, "SQS_PROOF_REGISTRATION_JOB_PROCESSING_QUEUE_URL"),
-        (PROOF_REGISTRATION_JOB_VERIFICATION_QUEUE, "SQS_PROOF_REGISTRATION_JOB_VERIFICATION_QUEUE_URL"),
-        (PROVING_JOB_PROCESSING_QUEUE, "SQS_PROVING_JOB_PROCESSING_QUEUE_URL"),
-        (PROVING_JOB_VERIFICATION_QUEUE, "SQS_PROVING_JOB_VERIFICATION_QUEUE_URL"),
-        (SNOS_JOB_PROCESSING_QUEUE, "SQS_SNOS_JOB_PROCESSING_QUEUE_URL"),
-        (SNOS_JOB_VERIFICATION_QUEUE, "SQS_SNOS_JOB_VERIFICATION_QUEUE_URL"),
-        (UPDATE_STATE_JOB_PROCESSING_QUEUE, "SQS_UPDATE_STATE_JOB_PROCESSING_QUEUE_URL"),
-        (UPDATE_STATE_JOB_VERIFICATION_QUEUE, "SQS_UPDATE_STATE_JOB_VERIFICATION_QUEUE_URL"),
-        (JOB_HANDLE_FAILURE_QUEUE, "SQS_JOB_HANDLE_FAILURE_QUEUE_URL"),
-        (WORKER_TRIGGER_QUEUE, "SQS_WORKER_TRIGGER_QUEUE_URL"),
-    ]);
+#[derive(Debug, Clone, Serialize)]
+pub struct AWSSQSValidatedArgs {
+    pub queue_base_url: String,
+    pub sqs_prefix: String,
+    pub sqs_suffix: String,
+}
+
+impl AWSSQSValidatedArgs {
+    pub fn get_queue_url(&self, queue_type: QueueType) -> String {
+        let name = format!("{}/{}", self.queue_base_url, self.get_queue_name(queue_type));
+        name
+    }
+
+    pub fn get_queue_name(&self, queue_type: QueueType) -> String {
+        format!("{}_{}_{}", self.sqs_prefix, queue_type, self.sqs_suffix)
```

Comment:
can make this a queue value and not sqs

Reply:
Initially it was a part of `AWSSQSValidatedArgs` which was wrong, I have shifted it to be a part of `SqsQueue` for now, when we add another queue provider we can exercise this generalisation